hotness/anitya.py

```diff
@@ -215,10 +215,18 @@ def add_new_project(self, name, homepage):
         )
 
         # Try to guess at what backend to prefill...
-        for target, backend in backends.items():
-            if target in homepage:
+
+        # Start with package prefix first...
+        for prefix, backend in prefixes.items():
+            if name.startswith(prefix) and backend != None:
                 data['backend'] = backend
-                break
+
+        # And try homepage if backend was not detected based on prefix
+        if 'backend' not in data:
+            for target, backend in backends.items():
+                if target in homepage:
+                    data['backend'] = backend
+                    break
```

Comment:
Almost 100% gems are available on RubyGems, while probably 70% are hosted on GitHub and may be 10% of them, although hosted on GitHub has different homepage. So for RubyGems, trying homepage first is just waste of energy.

Also, the git tags are not standardized and people are using different formats, that is another downside of GitHub backend ...